app/models/discussion.rb

```diff
@@ -8,10 +8,6 @@ class Discussion < ActiveRecord::Base
 
   default_scope -> { order('updated_at DESC') }
 
-  after_save :update_subscriptions
```

Comment:
I tested that. doesn't seem to be happening. But I guess I can add a test for that, just to be sure